superset/sql_parse.py

```diff
@@ -35,7 +35,10 @@ def __init__(self, sql_statement):
         self._table_names = set()
         self._alias_names = set()
         self._limit = None
-
+        # Start Dremio Support
+        self.sql = self.sql.replace('[','"')
+        self.sql = self.sql.replace(']','"')
+        # End Dremio Support
```

Comment:
This should not be here, and as stated above any engine specifics should be moved to `db_engine_specs.py`.  It appears that the dialect that is rendering queries is incorrectly using square brackets where they should be double quotes? To me this would indicate that you've specified an incorrect dialect in your SQLAlchemy URI (e.g. `oracle+cx_oracle://` means oracle dialect with cx_oracle connector), which is causing the incorrect column quotes to be used.